openshift/ups-template.json

```diff
@@ -0,0 +1,529 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "redhat-mobile-backend",
+    "annotations": {
+      "description": "An example Mobile Backend using RedHat, JBoss, AeroGear, and FeedHenry technologies",
```

Comment:
In the `description`, what is the FeedHenry part in this template?

Also, regarding the _name_, I think I'd have `name`d it like _aerogear-unified-push_